README.md
[![codecov](https://codecov.io/gh/Mastersam07/cocoicons/branch/master/graph/badge.svg)](https://codecov.io/gh/Mastersam07/cocoicons)
[![Test](https://github.com/Mastersam07/cocoicons/actions/workflows/test.yaml/badge.svg?branch=master)](https://github.com/Mastersam07/cocoicons/actions/workflows/test.yaml)
[![pub package](https://img.shields.io/pub/v/cocoicons.svg?color=success&style=flat-square)](https://pub.dartlang.org/packages/cocoicons)
[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-success.svg?style=flat-square)](https://github.com/Mastersam07/cocoicons/pulls)

# cocoicons

The COCO icon pack with - +600 Editable icons available for your flutter project.

*Made from [COCO icon pack](https://www.figma.com/community/file/944228750903853832).*

## 🎖 Installation

In the `dependencies:` section of your _pubspec.yaml_, add the following line:

```yaml
cocoicons:
```

## 🎮 Usage

```dart
import 'package:cocoicons/cocoicons.dart';

class MyAwesomeWidget extends StatelessWidget {
  Widget build(BuildContext context) {
    return Card(
        child: Column(
            mainAxisAlignment: MainAxisAlignment.center,
            children: <Widget>[
                Icon(CocoIconLine.Notification, size: 50.0),
                const SizedBox(height: 30.0),
                Text('Line Notification Icon', textAlign: TextAlign.center),
            ],
        ),
    );
  }
}
```

## Example

View the flutter app in the `example` directory

## Screenshot
<img src="https://github.com/Mastersam07/cocoicons/raw/master/screenshots/1.png" width="400">

## 🐛 Bugs/Requests

If you encounter any problems feel free to open an issue. If you feel the library is
missing a feature, please raise a ticket on Github and I'll look into it.
Pull request are also welcome.

## 🤓 Developer(s)

[<img src="https://avatars3.githubusercontent.com/u/31275429?s=460&u=b935d608a06c1604bae1d971e69a731480a27d46&v=4" width="180" />](https://mastersam.tech)
#### **Abada Samuel Oghenero**
<p>
<a href="https://twitter.com/mastersam_"><img src="https://github.com/aritraroy/social-icons/blob/master/twitter-icon.png?raw=true" width="60"></a>
<a href="https://linkedin.com/in/abada-samuel/"><img src="https://github.com/aritraroy/social-icons/blob/master/linkedin-icon.png?raw=true" width="60"></a>
<a href="https://medium.com/@sammytech"><img src="https://github.com/aritraroy/social-icons/blob/master/medium-icon.png?raw=true" width="60"></a>
<a href="https://facebook.com/abada.samueloghenero"><img src="https://github.com/aritraroy/social-icons/blob/master/facebook-icon.png?raw=true" width="60"></a>
</p>

CHANGELOG.md
## 0.1.0

* Line icons
* Bold icons

## 0.0.1

* Initial release.


test/cocoicons_test.dart
import 'package:cocoicons/cocoicons.dart';
import 'package:cocoicons/src/cocoicons_data.dart';
import 'package:flutter_test/flutter_test.dart';

void main() {
  test('Test that line icons are generated', () {
    String lineFamily = 'cocoline';
    CocoIconsData icon = CocoIconsData(0xe900, lineFamily);
    expect(icon.codePoint, 0xe900);
  });

  test('Test that bold icons are generated', () {
    String boldFamily = 'cocobold';
    CocoIconsData icon = CocoIconsData(0xe900, boldFamily);
    expect(icon.codePoint, 0xe900);
  });

  test('Test line icon', () {
    expect(CocoIconLine.Add.codePoint, 0xe900);
  });

  test('Test line icon family', () {
    String lineFamily = 'cocoline';
    expect(CocoIconLine.Add.fontFamily, lineFamily);
  });

  test('Test icon package name', () {
    expect(CocoIconLine.Add.fontPackage, 'cocoicons');
  });
}


example/lib/main.dart
import 'package:cocoicons/cocoicons.dart';
import 'package:example/icon_model.dart';
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Cocoicons Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: const MyHomePage(title: 'Cocoicons Demo'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key? key, required this.title}) : super(key: key);

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  List<IconModel> sampleIcons = [
    IconModel(icon: CocoIconLine.Add, name: "Line Add icon"),
    IconModel(icon: CocoIconBold.Add, name: "Bold Add icon"),
    IconModel(icon: CocoIconLine.Notification, name: "Line Notification icon"),
    IconModel(icon: CocoIconBold.Notification, name: "Bold Notification icon"),
    IconModel(
        icon: CocoIconLine.Notification_2, name: "Line Notification icon"),
    IconModel(
        icon: CocoIconBold.Notification_2, name: "Bold Notification icon"),
  ];
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: GridView.builder(
        padding: const EdgeInsets.symmetric(horizontal: 10.0),
        gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(
          crossAxisCount: 2,
          crossAxisSpacing: 8.0,
        ),
        itemBuilder: (BuildContext context, int index) {
          return Card(
            child: Column(
              mainAxisAlignment: MainAxisAlignment.center,
              children: <Widget>[
                Icon(sampleIcons[index].icon, size: 50.0),
                const SizedBox(height: 30.0),
                Text(sampleIcons[index].name, textAlign: TextAlign.center),
              ],
            ),
          );
        },
        itemCount: sampleIcons.length,
      ),
    );
  }
}


lib/src/cocoline_icons.dart
import 'package:flutter/widgets.dart';

import 'cocoicons_data.dart';

class CocoIconLine {
  CocoIconLine._();

  static const String _fontFamily = 'cocoline';

  static const IconData Add = CocoIconsData(0xe900, _fontFamily);
  static const IconData Arrow_Bottom_1 = CocoIconsData(0xe901, _fontFamily);
  static const IconData Arrow_Bottom_2 = CocoIconsData(0xe902, _fontFamily);
  static const IconData Arrow_Bottom_3 = CocoIconsData(0xe903, _fontFamily);
  static const IconData Arrow_Bottom = CocoIconsData(0xe904, _fontFamily);
  static const IconData Arrow_Left_1 = CocoIconsData(0xe905, _fontFamily);
  static const IconData Arrow_Left_2 = CocoIconsData(0xe906, _fontFamily);
  static const IconData Arrow_Left_3 = CocoIconsData(0xe907, _fontFamily);
  static const IconData Arrow_Left = CocoIconsData(0xe908, _fontFamily);
  static const IconData Arrow_Right_1 = CocoIconsData(0xe909, _fontFamily);
  static const IconData Arrow_Right_2 = CocoIconsData(0xe90a, _fontFamily);
  static const IconData Arrow_Right_3 = CocoIconsData(0xe90b, _fontFamily);
  static const IconData Arrow_Right = CocoIconsData(0xe90c, _fontFamily);
  static const IconData Arrow_Swap = CocoIconsData(0xe90d, _fontFamily);
  static const IconData Arrow_Top_1 = CocoIconsData(0xe90e, _fontFamily);
  static const IconData Arrow_Top_2 = CocoIconsData(0xe90f, _fontFamily);
  static const IconData Arrow_Top_3 = CocoIconsData(0xe910, _fontFamily);
  static const IconData Arrow_Top = CocoIconsData(0xe911, _fontFamily);
  static const IconData Award = CocoIconsData(0xe912, _fontFamily);
  static const IconData Bag_2 = CocoIconsData(0xe913, _fontFamily);
  static const IconData Bag = CocoIconsData(0xe914, _fontFamily);
  static const IconData Battery = CocoIconsData(0xe915, _fontFamily);
  static const IconData Calendar = CocoIconsData(0xe916, _fontFamily);
  static const IconData Call_Calling = CocoIconsData(0xe917, _fontFamily);
  static const IconData Call_Muted = CocoIconsData(0xe918, _fontFamily);
  static const IconData Call_Slash = CocoIconsData(0xe919, _fontFamily);
  static const IconData Camera_Slash = CocoIconsData(0xe91a, _fontFamily);
  static const IconData Camera = CocoIconsData(0xe91b, _fontFamily);
  static const IconData Card = CocoIconsData(0xe91c, _fontFamily);
  static const IconData Charge = CocoIconsData(0xe91d, _fontFamily);
  static const IconData Chart = CocoIconsData(0xe91e, _fontFamily);
  static const IconData Check = CocoIconsData(0xe91f, _fontFamily);
  static const IconData Checklist = CocoIconsData(0xe920, _fontFamily);
  static const IconData Clock = CocoIconsData(0xe921, _fontFamily);
  static const IconData Close = CocoIconsData(0xe922, _fontFamily);
  static const IconData Cloud_Drizzle = CocoIconsData(0xe923, _fontFamily);
  static const IconData Cloud_Lightning = CocoIconsData(0xe924, _fontFamily);
  static const IconData Cloud_1 = CocoIconsData(0xe925, _fontFamily);
  static const IconData Cloud = CocoIconsData(0xe926, _fontFamily);
  static const IconData Comparison = CocoIconsData(0xe927, _fontFamily);
  static const IconData Copy = CocoIconsData(0xe928, _fontFamily);
  static const IconData CPU = CocoIconsData(0xe929, _fontFamily);
  static const IconData Cup = CocoIconsData(0xe92a, _fontFamily);
  static const IconData Discovery = CocoIconsData(0xe92b, _fontFamily);
  static const IconData Document = CocoIconsData(0xe92c, _fontFamily);
  static const IconData Dollar = CocoIconsData(0xe92d, _fontFamily);
  static const IconData Edit_2 = CocoIconsData(0xe92e, _fontFamily);
  static const IconData Edit_3 = CocoIconsData(0xe92f, _fontFamily);
  static const IconData Edit_1 = CocoIconsData(0xe930, _fontFamily);
  static const IconData Edit = CocoIconsData(0xe931, _fontFamily);
  static const IconData Export = CocoIconsData(0xe932, _fontFamily);
  static const IconData Eye_slash = CocoIconsData(0xe933, _fontFamily);
  static const IconData Eye = CocoIconsData(0xe934, _fontFamily);
  static const IconData Filter = CocoIconsData(0xe935, _fontFamily);
  static const IconData Folder = CocoIconsData(0xe936, _fontFamily);
  static const IconData Gallery = CocoIconsData(0xe937, _fontFamily);
  static const IconData GPS = CocoIconsData(0xe938, _fontFamily);
  static const IconData Grid_2 = CocoIconsData(0xe939, _fontFamily);
  static const IconData Grid_3 = CocoIconsData(0xe93a, _fontFamily);
  static const IconData Grid = CocoIconsData(0xe93b, _fontFamily);
  static const IconData Headphone_slash = CocoIconsData(0xe93c, _fontFamily);
  static const IconData Headphone = CocoIconsData(0xe93d, _fontFamily);
  static const IconData Heart = CocoIconsData(0xe93e, _fontFamily);
  static const IconData Home_2 = CocoIconsData(0xe93f, _fontFamily);
  static const IconData Home = CocoIconsData(0xe940, _fontFamily);
  static const IconData Info = CocoIconsData(0xe941, _fontFamily);
  static const IconData Instagram = CocoIconsData(0xe942, _fontFamily);
  static const IconData Loading_1 = CocoIconsData(0xe943, _fontFamily);
  static const IconData Loading = CocoIconsData(0xe944, _fontFamily);
  static const IconData Location = CocoIconsData(0xe945, _fontFamily);
  static const IconData Lock_1 = CocoIconsData(0xe946, _fontFamily);
  static const IconData Lock = CocoIconsData(0xe947, _fontFamily);
  static const IconData Marking = CocoIconsData(0xe948, _fontFamily);
  static const IconData Menu_1 = CocoIconsData(0xe949, _fontFamily);
  static const IconData Menu = CocoIconsData(0xe94a, _fontFamily);
  static const IconData Message_2 = CocoIconsData(0xe94b, _fontFamily);
  static const IconData Message_3 = CocoIconsData(0xe94c, _fontFamily);
  static const IconData Message_4 = CocoIconsData(0xe94d, _fontFamily);
  static const IconData Message_5 = CocoIconsData(0xe94e, _fontFamily);
  static const IconData Message = CocoIconsData(0xe94f, _fontFamily);
  static const IconData Mic = CocoIconsData(0xe950, _fontFamily);
  static const IconData Minus = CocoIconsData(0xe951, _fontFamily);
  static const IconData Mobile = CocoIconsData(0xe952, _fontFamily);
  static const IconData Monitor = CocoIconsData(0xe953, _fontFamily);
  static const IconData Moon = CocoIconsData(0xe954, _fontFamily);
  static const IconData More_1 = CocoIconsData(0xe955, _fontFamily);
  static const IconData More = CocoIconsData(0xe956, _fontFamily);
  static const IconData Mouse = CocoIconsData(0xe957, _fontFamily);
  static const IconData Note_2 = CocoIconsData(0xe958, _fontFamily);
  static const IconData Note_3 = CocoIconsData(0xe959, _fontFamily);
  static const IconData Note_add = CocoIconsData(0xe95a, _fontFamily);
  static const IconData Note_check = CocoIconsData(0xe95b, _fontFamily);
  static const IconData Note_delete = CocoIconsData(0xe95c, _fontFamily);
  static const IconData Note_minus = CocoIconsData(0xe95d, _fontFamily);
  static const IconData Note = CocoIconsData(0xe95e, _fontFamily);
  static const IconData Notification_2 = CocoIconsData(0xe95f, _fontFamily);
  static const IconData Notification_3 = CocoIconsData(0xe960, _fontFamily);
  static const IconData Notification_1 = CocoIconsData(0xe961, _fontFamily);
  static const IconData Notification = CocoIconsData(0xe962, _fontFamily);
  static const IconData Phone = CocoIconsData(0xe963, _fontFamily);
  static const IconData Play_1 = CocoIconsData(0xe964, _fontFamily);
  static const IconData Play = CocoIconsData(0xe965, _fontFamily);
  static const IconData Plus = CocoIconsData(0xe966, _fontFamily);
  static const IconData Presentation_1 = CocoIconsData(0xe967, _fontFamily);
  static const IconData Presentation = CocoIconsData(0xe968, _fontFamily);
  static const IconData Programming = CocoIconsData(0xe969, _fontFamily);
  static const IconData Question = CocoIconsData(0xe96a, _fontFamily);
  static const IconData Report = CocoIconsData(0xe96b, _fontFamily);
  static const IconData Reserve = CocoIconsData(0xe96c, _fontFamily);
  static const IconData Rotate_Left = CocoIconsData(0xe96d, _fontFamily);
  static const IconData Rotate_Right = CocoIconsData(0xe96e, _fontFamily);
  static const IconData Saved = CocoIconsData(0xe96f, _fontFamily);
  static const IconData Scan_2 = CocoIconsData(0xe970, _fontFamily);
  static const IconData Scan_3 = CocoIconsData(0xe971, _fontFamily);
  static const IconData Scan = CocoIconsData(0xe972, _fontFamily);
  static const IconData Search = CocoIconsData(0xe973, _fontFamily);
  static const IconData Send = CocoIconsData(0xe974, _fontFamily);
  static const IconData Setting_2 = CocoIconsData(0xe975, _fontFamily);
  static const IconData Setting_3 = CocoIconsData(0xe976, _fontFamily);
  static const IconData Setting = CocoIconsData(0xe977, _fontFamily);
  static const IconData Shield_check = CocoIconsData(0xe978, _fontFamily);
  static const IconData Shield_vulnerable = CocoIconsData(0xe979, _fontFamily);
  static const IconData Shield = CocoIconsData(0xe97a, _fontFamily);
  static const IconData Shopping_cart = CocoIconsData(0xe97b, _fontFamily);
  static const IconData Sort = CocoIconsData(0xe97c, _fontFamily);
  static const IconData Star_1 = CocoIconsData(0xe97d, _fontFamily);
  static const IconData Star = CocoIconsData(0xe97e, _fontFamily);
  static const IconData Status = CocoIconsData(0xe97f, _fontFamily);
  static const IconData Sun = CocoIconsData(0xe980, _fontFamily);
  static const IconData Sunrise = CocoIconsData(0xe981, _fontFamily);
  static const IconData Text = CocoIconsData(0xe982, _fontFamily);
  static const IconData Tick = CocoIconsData(0xe983, _fontFamily);
  static const IconData Todo = CocoIconsData(0xe984, _fontFamily);
  static const IconData Trash_2 = CocoIconsData(0xe985, _fontFamily);
  static const IconData Trash = CocoIconsData(0xe986, _fontFamily);
  static const IconData Turn_off = CocoIconsData(0xe987, _fontFamily);
  static const IconData TV = CocoIconsData(0xe988, _fontFamily);
  static const IconData Two_user = CocoIconsData(0xe989, _fontFamily);
  static const IconData Unlock = CocoIconsData(0xe98a, _fontFamily);
  static const IconData User_plus = CocoIconsData(0xe98b, _fontFamily);
  static const IconData User_scan = CocoIconsData(0xe98c, _fontFamily);
  static const IconData User = CocoIconsData(0xe98d, _fontFamily);
  static const IconData Video = CocoIconsData(0xe98e, _fontFamily);
  static const IconData Volume_Cross = CocoIconsData(0xe98f, _fontFamily);
  static const IconData Volume_High = CocoIconsData(0xe990, _fontFamily);
  static const IconData Volume_Low = CocoIconsData(0xe991, _fontFamily);
  static const IconData Volume_Mute = CocoIconsData(0xe992, _fontFamily);
  static const IconData Volume_Slash = CocoIconsData(0xe993, _fontFamily);
  static const IconData Wallet = CocoIconsData(0xe994, _fontFamily);
  static const IconData Work = CocoIconsData(0xe995, _fontFamily);
  static const IconData Zoom_In = CocoIconsData(0xe996, _fontFamily);
  static const IconData Zoom_Out = CocoIconsData(0xe997, _fontFamily);
}


lib/src/cocobold_icons.dart
import 'package:flutter/widgets.dart';

import 'cocoicons_data.dart';

class CocoIconBold {
  CocoIconBold._();

  static const String _fontFamily = 'cocobold';

  static const IconData Add = CocoIconsData(0xe900, _fontFamily);
  static const IconData Arrow_Bottom_1 = CocoIconsData(0xe901, _fontFamily);
  static const IconData Arrow_Bottom_2 = CocoIconsData(0xe902, _fontFamily);
  static const IconData Arrow_Bottom_3 = CocoIconsData(0xe903, _fontFamily);
  static const IconData Arrow_Bottom = CocoIconsData(0xe904, _fontFamily);
  static const IconData Arrow_Left_1 = CocoIconsData(0xe905, _fontFamily);
  static const IconData Arrow_Left_2 = CocoIconsData(0xe906, _fontFamily);
  static const IconData Arrow_Left_3 = CocoIconsData(0xe907, _fontFamily);
  static const IconData Arrow_Left = CocoIconsData(0xe908, _fontFamily);
  static const IconData Arrow_Right_1 = CocoIconsData(0xe909, _fontFamily);
  static const IconData Arrow_Right_2 = CocoIconsData(0xe90a, _fontFamily);
  static const IconData Arrow_Right_3 = CocoIconsData(0xe90b, _fontFamily);
  static const IconData Arrow_Right = CocoIconsData(0xe90c, _fontFamily);
  static const IconData Arrow_Swap = CocoIconsData(0xe90d, _fontFamily);
  static const IconData Arrow_Top_1 = CocoIconsData(0xe90e, _fontFamily);
  static const IconData Arrow_Top_2 = CocoIconsData(0xe90f, _fontFamily);
  static const IconData Arrow_Top_3 = CocoIconsData(0xe910, _fontFamily);
  static const IconData Arrow_Top = CocoIconsData(0xe911, _fontFamily);
  static const IconData Award = CocoIconsData(0xe912, _fontFamily);
  static const IconData Bag_2 = CocoIconsData(0xe913, _fontFamily);
  static const IconData Bag = CocoIconsData(0xe914, _fontFamily);
  static const IconData Battery = CocoIconsData(0xe915, _fontFamily);
  static const IconData Calendar = CocoIconsData(0xe916, _fontFamily);
  static const IconData Call_Calling = CocoIconsData(0xe917, _fontFamily);
  static const IconData Call_Muted = CocoIconsData(0xe918, _fontFamily);
  static const IconData Call_Slash = CocoIconsData(0xe919, _fontFamily);
  static const IconData Camera_Slash = CocoIconsData(0xe91a, _fontFamily);
  static const IconData Camera = CocoIconsData(0xe91b, _fontFamily);
  static const IconData Card = CocoIconsData(0xe91c, _fontFamily);
  static const IconData Charge = CocoIconsData(0xe91d, _fontFamily);
  static const IconData Chart = CocoIconsData(0xe91e, _fontFamily);
  static const IconData Check = CocoIconsData(0xe91f, _fontFamily);
  static const IconData Checklist = CocoIconsData(0xe920, _fontFamily);
  static const IconData Clock = CocoIconsData(0xe921, _fontFamily);
  static const IconData Close = CocoIconsData(0xe922, _fontFamily);
  static const IconData Cloud_Drizzle = CocoIconsData(0xe923, _fontFamily);
  static const IconData Cloud_Lightning = CocoIconsData(0xe924, _fontFamily);
  static const IconData Cloud_1 = CocoIconsData(0xe925, _fontFamily);
  static const IconData Cloud = CocoIconsData(0xe926, _fontFamily);
  static const IconData Comparison = CocoIconsData(0xe927, _fontFamily);
  static const IconData Copy = CocoIconsData(0xe928, _fontFamily);
  static const IconData CPU = CocoIconsData(0xe929, _fontFamily);
  static const IconData Cup = CocoIconsData(0xe92a, _fontFamily);
  static const IconData Discovery = CocoIconsData(0xe92b, _fontFamily);
  static const IconData Document = CocoIconsData(0xe92c, _fontFamily);
  static const IconData Dollar = CocoIconsData(0xe92d, _fontFamily);
  static const IconData Edit_2 = CocoIconsData(0xe92e, _fontFamily);
  static const IconData Edit_3 = CocoIconsData(0xe92f, _fontFamily);
  static const IconData Edit_1 = CocoIconsData(0xe930, _fontFamily);
  static const IconData Edit = CocoIconsData(0xe931, _fontFamily);
  static const IconData Export = CocoIconsData(0xe932, _fontFamily);
  static const IconData Eye_Slash = CocoIconsData(0xe933, _fontFamily);
  static const IconData Eye = CocoIconsData(0xe934, _fontFamily);
  static const IconData Filter = CocoIconsData(0xe935, _fontFamily);
  static const IconData Folder = CocoIconsData(0xe936, _fontFamily);
  static const IconData Gallery = CocoIconsData(0xe937, _fontFamily);
  static const IconData GPS = CocoIconsData(0xe938, _fontFamily);
  static const IconData Grid_2 = CocoIconsData(0xe939, _fontFamily);
  static const IconData Grid_3 = CocoIconsData(0xe93a, _fontFamily);
  static const IconData Grid = CocoIconsData(0xe93b, _fontFamily);
  static const IconData Headphone_slash = CocoIconsData(0xe93c, _fontFamily);
  static const IconData Headphone = CocoIconsData(0xe93d, _fontFamily);
  static const IconData Heart = CocoIconsData(0xe93e, _fontFamily);
  static const IconData Home_2 = CocoIconsData(0xe93f, _fontFamily);
  static const IconData Home = CocoIconsData(0xe940, _fontFamily);
  static const IconData Info = CocoIconsData(0xe941, _fontFamily);
  static const IconData Insta = CocoIconsData(0xe942, _fontFamily);
  static const IconData Loading_1 = CocoIconsData(0xe943, _fontFamily);
  static const IconData Loading = CocoIconsData(0xe944, _fontFamily);
  static const IconData Location = CocoIconsData(0xe945, _fontFamily);
  static const IconData Lock_1 = CocoIconsData(0xe946, _fontFamily);
  static const IconData Lock = CocoIconsData(0xe947, _fontFamily);
  static const IconData Marking = CocoIconsData(0xe948, _fontFamily);
  static const IconData Menu_1 = CocoIconsData(0xe949, _fontFamily);
  static const IconData Menu = CocoIconsData(0xe94a, _fontFamily);
  static const IconData Message_2 = CocoIconsData(0xe94b, _fontFamily);
  static const IconData Message_3 = CocoIconsData(0xe94c, _fontFamily);
  static const IconData Message_4 = CocoIconsData(0xe94d, _fontFamily);
  static const IconData Message_5 = CocoIconsData(0xe94e, _fontFamily);
  static const IconData Message = CocoIconsData(0xe94f, _fontFamily);
  static const IconData Mic = CocoIconsData(0xe950, _fontFamily);
  static const IconData Minus = CocoIconsData(0xe951, _fontFamily);
  static const IconData Mobile = CocoIconsData(0xe952, _fontFamily);
  static const IconData Monitor = CocoIconsData(0xe953, _fontFamily);
  static const IconData Moon_1 = CocoIconsData(0xe954, _fontFamily);
  static const IconData Moon = CocoIconsData(0xe955, _fontFamily);
  static const IconData More_Circle = CocoIconsData(0xe956, _fontFamily);
  static const IconData More = CocoIconsData(0xe957, _fontFamily);
  static const IconData Mouse = CocoIconsData(0xe958, _fontFamily);
  static const IconData Note_2 = CocoIconsData(0xe959, _fontFamily);
  static const IconData Note_3 = CocoIconsData(0xe95a, _fontFamily);
  static const IconData Note_add = CocoIconsData(0xe95b, _fontFamily);
  static const IconData Note_check = CocoIconsData(0xe95c, _fontFamily);
  static const IconData Note_delete = CocoIconsData(0xe95d, _fontFamily);
  static const IconData Note_minus = CocoIconsData(0xe95e, _fontFamily);
  static const IconData Note = CocoIconsData(0xe95f, _fontFamily);
  static const IconData Notification_2 = CocoIconsData(0xe960, _fontFamily);
  static const IconData Notification_3 = CocoIconsData(0xe961, _fontFamily);
  static const IconData Notification_1 = CocoIconsData(0xe962, _fontFamily);
  static const IconData Notification = CocoIconsData(0xe963, _fontFamily);
  static const IconData Phone = CocoIconsData(0xe964, _fontFamily);
  static const IconData Play_1 = CocoIconsData(0xe965, _fontFamily);
  static const IconData Play = CocoIconsData(0xe966, _fontFamily);
  static const IconData Plus = CocoIconsData(0xe967, _fontFamily);
  static const IconData Presentation_1 = CocoIconsData(0xe968, _fontFamily);
  static const IconData Presentation = CocoIconsData(0xe969, _fontFamily);
  static const IconData Programming = CocoIconsData(0xe96a, _fontFamily);
  static const IconData Question = CocoIconsData(0xe96b, _fontFamily);
  static const IconData Report = CocoIconsData(0xe96c, _fontFamily);
  static const IconData Reserve = CocoIconsData(0xe96d, _fontFamily);
  static const IconData Rotate_Left = CocoIconsData(0xe96e, _fontFamily);
  static const IconData Rotate_Right = CocoIconsData(0xe96f, _fontFamily);
  static const IconData Saved = CocoIconsData(0xe970, _fontFamily);
  static const IconData Scan_2 = CocoIconsData(0xe971, _fontFamily);
  static const IconData Scan_3 = CocoIconsData(0xe972, _fontFamily);
  static const IconData Scan = CocoIconsData(0xe973, _fontFamily);
  static const IconData Search = CocoIconsData(0xe974, _fontFamily);
  static const IconData Send = CocoIconsData(0xe975, _fontFamily);
  static const IconData Setting_2 = CocoIconsData(0xe976, _fontFamily);
  static const IconData Setting_3 = CocoIconsData(0xe977, _fontFamily);
  static const IconData Setting = CocoIconsData(0xe978, _fontFamily);
  static const IconData Shield_check = CocoIconsData(0xe979, _fontFamily);
  static const IconData Shield_vulnerable = CocoIconsData(0xe97a, _fontFamily);
  static const IconData Shield = CocoIconsData(0xe97b, _fontFamily);
  static const IconData Shopping_Cart = CocoIconsData(0xe97c, _fontFamily);
  static const IconData Sort = CocoIconsData(0xe97d, _fontFamily);
  static const IconData Star = CocoIconsData(0xe97e, _fontFamily);
  static const IconData Status = CocoIconsData(0xe97f, _fontFamily);
  static const IconData Sun = CocoIconsData(0xe980, _fontFamily);
  static const IconData Sunrise = CocoIconsData(0xe981, _fontFamily);
  static const IconData Text = CocoIconsData(0xe982, _fontFamily);
  static const IconData Tick = CocoIconsData(0xe983, _fontFamily);
  static const IconData Todo = CocoIconsData(0xe984, _fontFamily);
  static const IconData Trash_2 = CocoIconsData(0xe985, _fontFamily);
  static const IconData Trash = CocoIconsData(0xe986, _fontFamily);
  static const IconData Turn_off = CocoIconsData(0xe987, _fontFamily);
  static const IconData TV = CocoIconsData(0xe988, _fontFamily);
  static const IconData Two_user = CocoIconsData(0xe989, _fontFamily);
  static const IconData Unlock = CocoIconsData(0xe98a, _fontFamily);
  static const IconData User_Scan = CocoIconsData(0xe98b, _fontFamily);
  static const IconData User_plus = CocoIconsData(0xe98c, _fontFamily);
  static const IconData User = CocoIconsData(0xe98d, _fontFamily);
  static const IconData Video = CocoIconsData(0xe98e, _fontFamily);
  static const IconData Volume_Cross = CocoIconsData(0xe98f, _fontFamily);
  static const IconData Volume_High = CocoIconsData(0xe990, _fontFamily);
  static const IconData Volume_Low = CocoIconsData(0xe991, _fontFamily);
  static const IconData Volume_Mute = CocoIconsData(0xe992, _fontFamily);
  static const IconData Volume_Slash = CocoIconsData(0xe993, _fontFamily);
  static const IconData Wallet = CocoIconsData(0xe994, _fontFamily);
  static const IconData Work = CocoIconsData(0xe995, _fontFamily);
  static const IconData Zoom_In = CocoIconsData(0xe996, _fontFamily);
  static const IconData Zoom_Out = CocoIconsData(0xe997, _fontFamily);
}
